tests/observability/metrics/conftest.py

```diff
@@ -872,17 +873,31 @@ def vm_memory_working_set_bytes(vm_for_test):
         bitmath.parse_string_unsafe(
             re.search(
                 r"\b(\d+Mi)\b",
-                run_command(
-                    command=shlex.split(
-                        f"oc adm top pod {vm_for_test.vmi.virt_launcher_pod.name} -n {vm_for_test.namespace}"
-                    ),
-                    check=False,
-                )[1],
+                vm_memory_working_set_command_output(vm=vm_for_test)[1],
             ).group(1)
         ).bytes
     )
 
 
+def vm_memory_working_set_command_output(vm: VirtualMachineForTests):
```

Comment:
This is not correct fix. You need to fix running_vm() call to ensure you wait for wait_for_cloud_init().